Two new commits since I last looked, and both close the gaps from netcode#161: `1eecd8af` wires the checkers into CI, and `aeb982cb` drives `CONNECTING -> ERROR` instead of only the happy path. The second one is the harder half — a state-machine checker that only ever sees a clean lifecycle is a table nothing observes.

**`STANDARD.md + STATE-MACHINE.md conformance` is red on `aeb982cb`, and it's a real bug in the checker, not in yojimbo.** The other 12 checks are green.

The failure, from the job log:

```
build failed — is the library built in build?
include/yojimbo.h:36:10: fatal error: serialize.h: No such file or directory
```

It is not what that message says. The library built fine (`[100%] Built target test`, immediately above). What failed is the *driver's own* compile, inside `verify_state_machine.py`:

```python
cmd = [cxx, "-std=c++11", "-I" + os.path.join(ROOT, "include"), "-I" + ROOT,
       "-I" + os.path.join(ROOT, "..", "serialize"),
       "-I" + os.path.join(ROOT, "..", "netcode"),
       "-I" + os.path.join(ROOT, "..", "reliable"),
```

`ROOT` is the repo root, and serialize/netcode/reliable are vendored *inside* it — `CMakeLists.txt:96-98` adds `${CMAKE_CURRENT_SOURCE_DIR}/{netcode,reliable,serialize}`, and `serialize/serialize.h` is there at this sha. So the three dep includes point one directory too high: `yojimbo/../serialize`. On a dev box with the sibling repos checked out beside yojimbo that resolves anyway, which is why it never showed. On a runner with only yojimbo checked out, it can't. Fix is dropping the `".."` from those three — `os.path.join(ROOT, "serialize")` and friends — matching what `-I ROOT/tlsf` two lines down already does correctly.

Worth noting what just happened, because it's your own argument landing: `verify_state_machine.py` had never actually executed in CI until `1eecd8af`, and it was broken the whole time. The first run of a check is also the first test *of* the check. `drive_error_paths.cpp` hasn't run yet either — the state-machine step dies before reaching it, so the new error-path coverage is still unverified.

Second, smaller: `build_and_run` reports every non-zero compile as "is the library built in `build`?", which sent me looking at the link step first. The missing-header case isn't a missing library. Cheap to say "could not compile the driver" and let stderr speak.

Not merging — yours.

--- a/tools/conformance/verify_state_machine.py
+++ b/tools/conformance/verify_state_machine.py
@@ -38,9 +38,9 @@ def build_and_run(cxx, build_dir, src_name="drive_state_machine.cpp"):
     with tempfile.TemporaryDirectory() as tmp:
         exe = os.path.join(tmp, "drive")
         cmd = [cxx, "-std=c++11", "-I" + os.path.join(ROOT, "include"), "-I" + ROOT,
-               "-I" + os.path.join(ROOT, "..", "serialize"),
-               "-I" + os.path.join(ROOT, "..", "netcode"),
-               "-I" + os.path.join(ROOT, "..", "reliable"),
+               "-I" + os.path.join(ROOT, "serialize"),
+               "-I" + os.path.join(ROOT, "netcode"),
+               "-I" + os.path.join(ROOT, "reliable"),
                "-I" + os.path.join(ROOT, "tlsf"),
                "-o", exe, src, "-L" + build_dir] + libs
         r = subprocess.run(cmd, capture_output=True, text=True)

